x-pack/test/api_integration/apis/cloud_security_posture/update_rules_config.ts

```diff
@@ -5,83 +5,87 @@
+    .then<PackageInfo>((response) => response.body.item);
+
+const getCspRulesSO = (supertest: SuperTest<Test>) =>
+  supertest
+    .get(`/api/saved_objects/_find?type=${CSP_RULE_SAVED_OBJECT_TYPE}`)
+    .expect(200)
+    .then<SavedObjectsFindResponse<CspRule>>((response) => response.body);
 
 export default function ({ getService }: FtrProviderContext) {
   const supertest = getService('supertest');
   const esArchiver = getService('esArchiver');
-  const chance = new Chance();
-  const kibanaServer = getService('kibanaServer');
-
-  // Failing: See https://github.com/elastic/kibana/issues/139683
-  describe.skip('POST /internal/cloud_security_posture/update_rules_config', () => {
-    let agentPolicyId: string;
 
+  describe('POST /internal/cloud_security_posture/update_rules_config', () => {
     before(async () => {
-      await kibanaServer.savedObjects.cleanStandardList();
       await esArchiver.load('x-pack/test/functional/es_archives/fleet/empty_fleet_server');
-
-      const { body: agentPolicyResponse } = await supertest
-        .post(`/api/fleet/agent_policies`)
-        .set('kbn-xsrf', 'xxxx')
-        .send({
-          name: 'Test policy',
-          namespace: 'default',
-        });
-      agentPolicyId = agentPolicyResponse.item.id;
+      await esArchiver.load('x-pack/test/functional/es_archives/fleet/agents');
     });
 
     after(async () => {
-      await kibanaServer.savedObjects.cleanStandardList();
       await esArchiver.unload('x-pack/test/functional/es_archives/fleet/empty_fleet_server');
+      await esArchiver.unload('x-pack/test/functional/es_archives/fleet/agents');
     });
 
-    it('returns a 404 response when package policy id does not exist', async () => {
-      const packagePolicyId = chance.guid();
+    it('updates package policy vars with new rules configuration', async () => {
+      const packagePolicy = await installIntegration(supertest);
+      const currentRules = await getCspRulesSO(supertest);
 
-      const { body: response } = await supertest
-        .post(`/internal/cloud_security_posture/update_rules_config`)
-        .set('kbn-xsrf', 'xxxx')
-        .send({
-          package_policy_id: packagePolicyId,
-          rules: [],
-        })
-        .expect(404);
-
-      expect(response.error).to.be('Not Found');
-    });
+      const ruleToDisable = currentRules.saved_objects[0];
+      const ruleToEnable = currentRules.saved_objects[1];
 
-    it(`Should return 200 for existing package policy id`, async () => {
-      const { body: postPackageResponse } = await supertest
-        .post(`/api/fleet/package_policies`)
+      const { body: updatedPackagePolicy } = await supertest
+        .post(UPDATE_RULES_CONFIG_ROUTE_PATH)
         .set('kbn-xsrf', 'xxxx')
         .send({
-          force: true,
-          name: 'cloud_security_posture-1',
-          description: '',
-          namespace: 'default',
-          policy_id: agentPolicyId,
-          enabled: true,
-          inputs: [],
```

Comment:
we need to pass the actual [inputs](https://github.com/elastic/integrations/blob/main/packages/cloud_security_posture/manifest.yml#L45-L51) we provide in our integration, as they are needed to compute the [streams](https://github.com/elastic/kibana/blob/8f51b55a31147462c1964e60c0bae40850b5a2bb/x-pack/plugins/fleet/server/services/agent_policies/package_policies_to_agent_inputs.ts#L63-L65) of agent policy which is how the configuration is eventually passed.  

seems like calling `packageToPackagePolicy(packageInfo)` is the easiest option